plugins/tiddlywiki/markdown/wrapper.js

```diff
@@ -94,7 +94,14 @@ function convertNodes(remarkableTree, isStartOfInline) {
 		var currentNode = remarkableTree[i];
 		switch (currentNode.type) {
 		case "paragraph_open":
-			i = wrappedElement("p", i, currentNode.level, "paragraph_close", remarkableTree);
+			// If the paragraph is a "tight" layout paragraph, don't wrap children in a <p> tag.
+			if (currentNode.tight) {
+				i = withChildren(i, currentNode.level, "paragraph_close", remarkableTree, function(children) {
+					out.push(...children);
```

Comment:
don't know whether the spread operator is allowed :)

Reply:
No ... spread operator isn't allowed. TW core only allows ES5 atm.

Reply:
Yes sorry about that, it's a nuisance but users find it useful to be able to run on older JS platforms.

Reply:
Gotcha -- replaced with `Array.prototype.push`.